demos/README.md

```diff
@@ -57,66 +57,65 @@ The table below shows the correlation between models, demos, and supported plugi
+| person-attributes-recognition-crossroad-0230     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                                                   | Supported | Supported | Supported   | Supported       |
+| person-attributes-recognition-crossroad-0234     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                                                   | Supported | Supported | Supported   |                 |
+| person-attributes-recognition-crossroad-0238     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                                                   | Supported | Supported | Supported   |                 |
+| person-detection-retail-0002                     | any demo that supports SSD\*-based models, above                                                             | Supported | Supported | Supported   | Supported       |
+| person-detection-retail-0013                     | any demo that supports SSD\*-based models, above                                                             | Supported | Supported | Supported   | Supported       |
+| person-reidentification-retail-0277              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                                                   | Supported | Supported |             |                 |
+| person-reidentification-retail-0286              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)<br>[Multi-Camera Multi-Target Tracking Demo](./python_demos/multi_camera_multi_target_tracking/README.md)    | Supported | Supported |       |        |
+| person-reidentification-retail-0287              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)<br>[Multi-Camera Multi-Target Tracking Demo](./python_demos/multi_camera_multi_target_tracking/README.md)    | Supported | Supported |       |        |
+| person-reidentification-retail-0288              | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)<br>[Multi-Camera Multi-Target Tracking Demo](./python_demos/multi_camera_multi_target_tracking/README.md)    | Supported | Supported |       |        |
+| person-vehicle-bike-detection-crossroad-0078     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                                                   | Supported | Supported | Supported   | Supported       |
+| person-vehicle-bike-detection-crossroad-1016     | [Crossroad Camera Demo](./crossroad_camera_demo/README.md)                                                   | Supported | Supported | Supported   |                 |
+| person-vehicle-bike-detection-crossroad-yolov3-1020 | [Object Detection for YOLO V3 Python\* Demo](./python_demos/object_detection_demo_yolov3_async/README.md) | Supported | Supported | Supported   |                 |
```

Comment:
This demo won't exist after #1744. I think it's best to wait until that PR is merged and then update this.

Reply:
agree